examples/simple/custom_contentview.py

```diff
@@ -3,7 +3,10 @@
 The content view API is explained in the mitmproxy.contentviews module.
 """
 from mitmproxy import contentviews
-from typing import Tuple, Iterable, AnyStr, List
+import typing
+
+
+CVIEWSWAPCASE = typing.Tuple[str, typing.Iterable[typing.List[typing.Tuple[str, typing.AnyStr]]]]
```

Comment:
This type annotations should be done here: https://github.com/mitmproxy/mitmproxy/blob/9d2350b6709532bf185c6e0f56342b7b780092c7/mitmproxy/contentviews/base.py#L17 and then re-used here. Please prefix types with `T`.